starfish/image/_filter/bandpass.py

```diff
@@ -31,6 +33,14 @@ def __init__(
             deviations (default 4)
         is_volume : bool
             If True, 3d (z, y, x) volumes will be filtered. By default, filter 2-d (y, x) planes
+        clip_method : int
```

Comment:
This is completely unintuitive. Please use an enum to denote the different clip methods.